timeboost-sequencer/src/lib.rs

```diff
@@ -282,7 +282,7 @@ impl Task {
                     secret,
                     b"dkg",
                 )?;
-                let bundle = DkgBundle::new(committee_id, ct, cm);
+                let bundle = DkgBundle::new(self.label, committee_id, ct, cm);
```

Comment:
Why does the `Sequencer` need to have a `DkgKeyStore`? And why can the decrypter not have a method `Decrypter::next_dkg_bundle(&self) -> Option<BundleVariant>` that we include if `some`?